vagrant/tutorial/bootstrap.py

```diff
@@ -24,11 +24,12 @@
     'fedora/zfs-release%s.noarch.rpm') % (rpm_dist,)
 check_call(['yum', 'install', '-y',  zfs_repo_url])
 
-clusterhq_repo_url = (
-    'https://storage.googleapis.com/'
-    'archive.clusterhq.com/'
+clusterhq_marketing_repo_url = (
```

Comment:
Given the above comments about the `clusterhq-release` package, we don't need to specify that this is the /marketing/ repo url.